contrib/gitian-build.py

```diff
@@ -135,7 +135,7 @@ def verify():
 def main():
     global args, workdir
 
-    parser = argparse.ArgumentParser(usage='%(prog)s [options] signer version')
+    parser = argparse.ArgumentParser(description='Script for running full Gitian builds.')
```

Comment:
15236 a force merge is not appropriate here. Please find what other backports may be needed.